docs/NOTICES.md

```diff
@@ -940,7 +941,7 @@
 
 | Field name          	| Description                                            	| Type   	|
 |---------------------	|--------------------------------------------------------	|--------	|
-| `csvRowNumber`      	| The row number of the faulty record.                   	| Long   	|
+| `stopsvRowNumber`     | The row number of the faulty record from `stops.txt`.    	| Long   	|
```

Comment:
stopCsvRowNumber

Such kinds of typos will be avoided when we generate this file automatically.